.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xbmc/addons/kodi-addon-dev-kit"]
+	path = xbmc/addons/kodi-addon-dev-kit
+	url = git://github.com/AlwinEsch/kodi-addon-dev-kit.git
```

Comment:
yes, and hence my initial comment. it's trivial for the cmake build system, for the others you have to do it manually supplied through depends on those platforms, through a separate build step as a bat file in visual studio (or through windows depends), manual rule in autotools.